cmd/internal/find/primary/size.go

```diff
@@ -3,35 +3,13 @@ package primary
 import (
 	"fmt"
 	"math"
-	"regexp"
-	"strconv"
 
 	"github.com/puppetlabs/wash/cmd/internal/find/grammar"
+	"github.com/puppetlabs/wash/cmd/internal/find/primary/numeric"
 	"github.com/puppetlabs/wash/cmd/internal/find/types"
 )
 
-var bytesMap = map[byte]uint64{
-	'c': 1,
-	'k': 1024,
-	'M': 1024 * 1024,
-	'G': 1024 * 1024 * 1024,
-	'T': 1024 * 1024 * 1024 * 1024,
-	'P': 1024 * 1024 * 1024 * 1024 * 1024,
-}
-
-// Use bytesOf to generate a more readable panic message
-func bytesOf(unit byte) uint64 {
-	if b, ok := bytesMap[unit]; ok {
-		return b
-	}
-	panic(fmt.Sprintf("cmdfind.bytesOf received an unexpected unit %v", unit))
-}
-
-var sizeValueRegex = regexp.MustCompile(`^(\+|-)?((\d+)|(\d+[ckMGTP]))$`)
-
-// sizePrimary => -size (+|-)?((\d+)|(\d+[ckMGTP]))
-//
-// where c => character (byte), k => kibibyte, M => mebibyte, G => gibibyte, T => tebibyte, P => pebibyte
+// sizePrimary => -size (+|-)?(\d+ | util.SizeRegex)
```

Comment:
Why does this say `util.SizeRegex`?

Reply:
Typo -- `numeric` was originally called `util`. Will fix that in the morning.